developer_guide/installation/system_requirements/system_install_osx_1011.rst

```diff
@@ -0,0 +1,276 @@
+System install on OS X 10.11 (and higher)
+=========================================
+
+Here is a quick guide to setup the :doc:`system_requirements` on OS X 10.11. This guide will help you to install all
+the packages and modules needed for Akeneo PIM, then configure it to match your local installation.
```

Comment:
"configure them" (if you are talking about the modules), or "configure the application" if you are talking about the PIM